v2v-helper/migrate/migrate.go

```diff
@@ -435,6 +457,98 @@ func (migobj *Migrate) CreateTargetInstance(vminfo vm.VMInfo) error {
 		return fmt.Errorf("failed to create VM: %s", err)
 	}
 	migobj.logMessage(fmt.Sprintf("VM created successfully: ID: %s", newVM.ID))
+
+	err = migobj.HealthCheck(ipaddresses)
+	if err != nil {
+		migobj.logMessage(fmt.Sprintf("Health Check failed: %s", err))
+	}
+
+	return nil
+}
+
+func (migobj *Migrate) pingVM(ips []string) error {
+	for _, ip := range ips {
+		migobj.logMessage(fmt.Sprintf("Pinging VM: %s", ip))
+		pinger, err := probing.NewPinger(ip)
+		if err != nil {
+			return fmt.Errorf("failed to create pinger: %s", err)
+		}
+		pinger.Count = 1
+		pinger.Timeout = time.Second * 10
+		err = pinger.Run()
+		if err != nil {
+			return fmt.Errorf("failed to run pinger: %s", err)
+		}
+		if pinger.Statistics().PacketLoss == 0 {
```

Comment:
I am a little concerned about this. Occasionally, when the mac address of the destination is not yet in the ARP table, the first packet is lost. Perhaps we modify the overall logic such that 4/5 reply packets must be received or some such variation, which accounts for 1 lost packet due to ARP lookup?